irohad/ametsuchi/wsv_query.hpp

```diff
@@ -106,17 +103,11 @@ namespace iroha {
 
       /**
        * Get accounts information from its key-value storage
-       * @param account_id
-       * @param creator_account_id
-       * @param detail
-       * @return
+       * @param account_id - account to get details about
+       * @return optional of account details
        */
-      virtual nonstd::optional<shared_model::interface::types::DetailType>
-      getAccountDetail(
-          const shared_model::interface::types::AccountIdType &account_id,
-          const shared_model::interface::types::AccountIdType
-              &creator_account_id,
-          const shared_model::interface::types::DetailType &detail) = 0;
+      virtual nonstd::optional<std::string> getAccountDetail(
```

Comment:
Maybe `boost::optional`? Please :((

Reply:
I didn't touch optional in this pull request. Also another methods here use nonstd::optional too. Better to fix this issue in a separate pull request I think.